parcellab/common/templates/_job.tpl

```diff
@@ -0,0 +1,27 @@
+{{- define "common.job" -}}
+{{- $job := default (dict "enabled" false) .job -}}
+{{- if not $job.job -}}
+{{- $_ := set $job "job" dict -}}
+{{- end -}}
+{{- if or .Values.job.enabled $job.enabled -}}
+{{- $fullname := include "common.fullname" . -}}
+{{- if $job.name -}}
+{{- $fullname = printf "%s-%s" $fullname $job.name -}}
+{{- end -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: {{ $fullname }}-
+  annotations:
```

Comment:
I see, so maybe the different in indentation is because the hook has to be a top level. I am not familiar with argo hooks, but if we want to different them from the Kubernetes spec, I would rename `job.job` to `job.jobSpec` or `job.jobTemplate`

Reply:
Otherwise, it is fine to have it in the same nesting level as we abstract it, in my opinion

Reply:
shit, 'fixed' cronjob instead, need to update again.